### PR TITLE
Gosseract update

### PR DESCRIPTION
**Pull request by mish15**
v1 gosseract disappeared. This fixes the dependency and odd use of goroutine. 

Tested and works. Didn't have time to write a test for this though. It needs one eventually

### CODE REVIEW

File: image_ocr.go [hunk 1]
```diff
@@ -25,22 +19,14 @@ func ConvertImage(r io.Reader) (string, map[string]string, error) {
...
+	client := gosseract.NewClient()
+	defer client.Close()
+	client.SetImage(f.Name())
+	text, err := client.Text()
+	if err != nil {
+		return "", nil, err
+	}
 
-// SetImageLanguages sets the languages parameter passed to gosseract.
-func SetImageLanguages(l string) {
```
Comment (jonathaningram) [inline image_ocr.go:42]: This changes the external package interface. Some users may be expecting to use this?

File: image_ocr.go [hunk 2]
```diff
@@ -25,22 +19,14 @@ func ConvertImage(r io.Reader) (string, map[string]string, error) {
 	defer f.Done()
 
 	meta := make(map[string]string)
-	out := make(chan string, 1)
-
-	// TODO: Why is this done in a separate goroutine when ConvertImage blocks until it returns?
-	go func(file *LocalFile) {
-		langs.RLock()
```
Comment (jonathaningram) [inline image_ocr.go:32]: It looks to me as if Gosseract would only work one call at a time per language, so we implemented locking which you've removed here (intentionally?). Not sure about why it was done in a goroutine though, would have to look into it.

Comment (mish15) [reply image_ocr.go:32]: yep cool, i put it back.

File: image_ocr.go (hunk 1 above)
Comment (mish15) [reply image_ocr.go:42]: agree, reinstated

**Comment by mish15** [comment]
@jonathaningram ping :)

**Pull request merged**